README.md

```diff
@@ -34,155 +37,253 @@ secure, and production-grade cloud infrastructure.
+  The number of nodes in the Kubernetes Engine cluster that will be used to run this environment. This field is supported for Cloud Composer environments in versions composer-1.*.*-airflow-*.*.*.
 
-  Specifies whether resources in the module will be created.
+- [**`software_config`**](#var-software_config): *(Optional `object(software_config)`)*<a name="var-software_config"></a>
 
-  Default is `true`.
+  The configuration settings for software inside the environment.
 
-- **`module_depends_on`**: _(Optional `list(dependencies)`)_
+  The object accepts the following attributes:
 
-  A list of dependencies. Any object can be _assigned_ to this list to define a hidden external dependency.
+  - [**`airflow_config_overrides`**](#attr-airflow_config_overrides-1): *(Optional `map(string)`)*<a name="attr-airflow_config_overrides-1"></a>
 
-  Example:
+    Apache Airflow configuration properties to override. Property keys contain the section and property names, separated by a hyphen, for example "core-dags_are_paused_at_creation".
+    
+    Section names must not contain hyphens ("-"), opening square brackets ("["), or closing square brackets ("]"). The property name must not be empty and cannot contain "=" or ";". Section and property names cannot contain characters: "." Apache Airflow configuration property names must be written in snake_case. Property values can contain any character, and can be written in any lower/upper case format. Certain Apache Airflow configuration property values are blacklisted, and cannot be overridden.
 
-  ```hcl
-  module_depends_on = [
-    google_network.network
-  ]
-  ```
+  - [**`pypi_packages`**](#attr-pypi_packages-1): *(Optional `map(string)`)*<a name="attr-pypi_packages-1"></a>
 
-#### Main Resource Configuration
+    Custom Python Package Index (PyPI) packages to be installed in the environment. Keys refer to the lowercase package name (e.g. "numpy"). Values are the lowercase extras and version specifier (e.g. "==1.12.0", "[devel,gcp_api]", "[devel]>=1.8.2, <1.9.2"). To specify a package without pinning it to a version specifier, use the empty string as the value.
 
-<!-- Example of a required variable:
+  - [**`env_variables`**](#attr-env_variables-1): *(Optional `map(string)`)*<a name="attr-env_variables-1"></a>
 
-- **`name`**: **_(Required `string`)_**
+    Additional environment variables to provide to the Apache Airflow scheduler, worker, and webserver processes. Environment variable names must match the regular expression [a-zA-Z_][a-zA-Z0-9_]*. They cannot specify Apache Airflow software configuration overrides (they cannot match the regular expression AIRFLOW__[A-Z0-9_]+__[A-Z0-9_]+), and they cannot match any of the following reserved names:
+    
+    AIRFLOW_HOME
+    C_FORCE_ROOT
+    CONTAINER_NAME
+    DAGS_FOLDER
+    GCP_PROJECT
+    GCS_BUCKET
+    GKE_CLUSTER_NAME
+    SQL_DATABASE
+    SQL_INSTANCE
+    SQL_PASSWORD
+    SQL_PROJECT
+    SQL_REGION
+    SQL_USER
```

Comment:
please separate by comma and use markdown formatting to make it more readable.

Reply:
Formatted